### ISSUE TITLE
VDB endpoint not working with application/x-www-form-urlencoded body (500 Internal Server Error)

### ISSUE THREAD

**Issue by simcof** [lionel-panhaleux/krcg-api#1]
To reproduce:
```
curl -X 'POST' \
  'https://api.krcg.org/vdb' \
  -H 'accept: application/json' \
  -H 'Content-Type: application/x-www-form-urlencoded' \
  -d 'url=https%3A%2F%2Fvdb.smeea.casa%2Fdecks%3Fid%3D2019r6vh'
```

Using application/json body instead generates the correct response, to compare:
```
curl -X 'POST' \
  'https://api.krcg.org/vdb' \
  -H 'accept: application/json' \
  -H 'Content-Type: application/json' \
  -d '{
  "url": "https://vdb.smeea.casa/decks?id=b798e734fff7404085f7b01ad2ccb479"
}'
```

**Comment by simcof** [comment]
After investigation, have identified issue is related to: https://github.com/lionel-panhaleux/krcg/issues/246

**Comment by lionel-panhaleux** [comment]
Fixed in KRCG 2.16, thanks for the PR!
